src/SourceBuild/tarball/content/repos/known-good.proj

```diff
@@ -18,7 +18,7 @@
         <RepositoryReference Include="runtime" />
       </ItemGroup>
     </When>
-    <When Condition="$(Platform.Contains('arm')) AND '$(BuildArchitecture)' != 'arm64'">
+    <When Condition="$(Platform.Contains('arm')) AND '$(BuildArchitecture)' != 'arm64' AND '$(BuildArchitecture)' != 'arm'">
```

Comment:
Agreed that this makes for kind of a weird condition but I'm not sure if there's no other possibilities either - maybe add a comment to that effect?

Reply:
Indeed - Idk why we do this cause when would `Platform.Contains('arm')` be true `BuildArchitecture =! {arm|arm64}`. Is it to cover `armhf`?

Reply:
There's also `armel` for Tizen now that I think about it, although source-build hasn't built that for ages.  I guess it's good to keep the options open though.

Reply:
This is when this piece of code was introduced: https://github.com/dotnet/source-build/pull/903/commits/3b449a74b15d836ca73d0079615f679c755dc207#diff-ced2f46c2f5a4c37221e805958a543b0e6f3fb6dd6674511bb97e7c265564a57R14. It seems like it was mean to capture arm64 only. The only arm architectures that are in the .NET release matrix are Arm32 and Arm64:

![image](https://user-images.githubusercontent.com/174451/163471161-3c7d6d66-27b2-4f64-8877-38f6365f3920.png)


I feel like we should change this to `Condition="'true' == 'false'"` to make sure Arm is captured by the `<Otherwise>` element, but leave it as an extension point for bringing up new architectures in the future.

Reply:
I'm good with this option with a comment to that effect.

Reply:
Implemented suggestion.